contracts/src/Summa.sol

```diff
@@ -85,6 +85,15 @@ contract Summa is Ownable {
         inclusionVerifier = _inclusionVerifier;
     }
 
+    /*
```

Comment:
What is this @alxkzmn @sifnoc?

Reply:
It's leftover from my audit comment, I will remove it.

Reply:
Removed [c575a29](https://github.com/summa-dev/summa-solvency/pull/179/commits/c575a299d5a585ea43204508a74541eda53af37e)